src/apiClients/AirTableApiClient.ts

```diff
@@ -1,20 +1,31 @@
-interface AirTableFilters {
+import Airtable from 'airtable';
+
+export interface AirTableFilters {
   interestCategories?: string[];
   riverSections?: string[];
 }
 
-interface Organization {
+export interface Organization {
   name: string;
   // fill in the rest
+  description: string;
+  activity: string;
+  interestCategories: string[];
+  riverSection: string;
+  phoneNumber: string;
+  url: string;
+  email: string;
 }
 
 class AirTableApiClient {
   private apiKey: string;
+  public base: any;
 
   constructor(apiKey: string) {
     this.apiKey = apiKey;
+    this.base = new Airtable({ apiKey: apiKey }).base('appEHr8iHguvEfXTQ');
     // remove this log when this API key is finally used somewhere
-    console.log(`SendGridAPIClient initialized with api key ${this.apiKey}`);
+    console.log(`AirTableAPIClient initialized with api key ${this.apiKey}`);
   }
 
   async getAllOrganizations(filters: AirTableFilters = {}): Promise<Organization[]> {
```

Comment:
Since we're just returning a promise, we can remove `async`

```suggestion
  getAllOrganizations(filters: AirTableFilters = {}): Promise<Organization[]> {
```